docs/queriying_documents.md

```diff
@@ -237,7 +237,7 @@ You can update a document by calling `save` on it:
 ```python
 movie = await Movie.query().get()
 
-movie.name = "Another Movie"
+movie.name = "  "
```

Comment:
Is this on purpose?

Reply:
No, I'm sorry, I don't know what happened, I'm going to return that change.